docs/next/guides/getting-started/setting-options.md

```diff
@@ -1,239 +1,631 @@
+    ```
+
+::: tip
+If you use Handsontable through [modules](@/guides/building-and-testing/modules.md): to use an option that comes from a Handsontable plugin, [import and register](@/guides/building-and-testing/modules.md#importing-plugins) that plugin when [initializing](@/guides/getting-started/installation.md#initialize-the-grid) your Handsontable instance.
+:::
 
-And finally, let's make one of the cells read-only.
+#### Example
 
-::: example #example4 --html 1 --css 2 --js 3
+In the example below, the [`cell`](@/api/options.md#cell) option sets cell `A1`(`0, 0`) and cell `B2`(`1, 1`) as [`readOnly`](@/api/options.md#readonly).
+
+The modified [`cell`](@/api/options.md#cell) options:
+- Overwrite the top-level [grid options](#setting-grid-options)
+- Overwrite the mid-level [column options](#setting-column-options)
+
+::: example #example4 --html 1 --js 2
 ```html
 <div id="example4" class="hot"></div>
 ```
-```css
-td.bg-read-only {
-  background-color: #f2f4fb;
-}
-```
 ```js
 const container = document.querySelector('#example4');
 
 const hot = new Handsontable(container, {
+  // top-level grid options that apply to the entire grid
   data: Handsontable.helper.createSpreadsheetData(5, 10),
-  cells(row, column) {
-    if (row === 1 && column === 1) {
-      return {
-        readOnly: true,
-        className: 'bg-read-only'
-      };
-    }
-  },
+  licenseKey: 'non-commercial-and-evaluation',
   width: 'auto',
   height: 'auto',
   rowHeaders: true,
   colHeaders: true,
-  licenseKey: 'non-commercial-and-evaluation'
+  readOnly: false,
+  cell: [
+    {
+      // bottom-level cell options overwrite the top-level grid options
+      // apply only to a cell with coordinates (0, 0)
+      row: 0,
+      col: 0,
+      readOnly: true,
+    },
+    {
+      // bottom-level cell options overwrite the top-level grid options
+      // apply only to a cell with coordinates (1, 1)
+      row: 1,
+      col: 1,
+      readOnly: true,
+    }
+  ]
 });
+
+// check a cell's options
+// returns `true`
+hot.getCellMeta(0, 0).readOnly;
+
+// returns `false`
+hot.getCellMeta(0, 1).readOnly;
 ```
 :::
 
-## `cells` vs `cell`
+### Checking cell options
 
-In the Options section of the API, you can find two similar options, [cell](@/api/options.md#cells-2) and [cell](@/api/options.md#cell). Using one of these options (or both), you can associate the custom properties or overwrite an entire table or column option for a particular cell. Both options allow you to set initial values for specific cells. You can check if the values are propagated by calling the [getCellMeta](@/api/core.md#getcellmeta) method. The method returns an object with all built-in options as well as those added by the developer.
+When Handsontable is running, you can check a cell's current options, using the [`getCellMeta()`](@/api/core.md#getcellmeta) method.
 
-#### `cell`
+The [`getCellMeta()`](@/api/core.md#getcellmeta) method returns an object with:
+- All built-in options
```

Comment:
We can mention here that built-in properties are stored in the `prototype` of CellMeta (https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object_prototypes).

Reply:
@mrpiotr-dev - done: 0efa67c